packages/server/src/migrations/functions/quotas1.ts

```diff
@@ -0,0 +1,20 @@
+import { runQuotaMigration } from "./usageQuotas"
```

Comment:
Just to check, why is this file called `quotas1.ts`? is there another file somewhere?

Reply:
It was just to put in place a pattern as we are going to need more quota migrations in future as we backfill new quotas (or fix existing ones)

Reply:
Not a big deal - but generally this structure is a bit more easy to grok:

`quotas_11_04_2021`